.github/workflows/ci.yml

```diff
@@ -152,6 +152,8 @@ jobs:
           # Some dependencies don't correctly specify a minimal version for their dependencies and will fail to build.
           # So we update these transitive dependencies here.
           cargo update tap faststr metainfo linkedbytes
+          # munge 0.4.2 will cause fail to use 1.7.1 so specify the correct version here.
+          cargo update -p munge --precise 0.4.1
```

Comment:
`munge 0.4.2` uses edition = 2024 now (which is 1.85). 
https://github.com/djkoloski/munge/commit/02e7be56edce3e9d64e5da86a5de7c476433818d

BTW we are discussing the MSRV policy here: https://lists.apache.org/thread/d10ntkzvkbyvh806063jblhhxgvczoth

Currently, datafusion is `1.82` https://github.com/apache/datafusion/blob/0bd9083a0b8bff6d261449a92dcd4d110976774a/Cargo.toml#L70

Therefore, I think pinning the `munge` version here is the best choice now.

Reply:
Thanks to refer this! Is it time to upgrade the version so that we don't need to pin the version? The last version is 7 month ago. also cc @liurenjie1024

Reply:
We can't use a version newer than datafusion, otherwise the datafusion integration will break.

Reply:
Thanks for reminding this!

Reply:
> We can't use a version newer than datafusion, otherwise the datafusion integration will break.

Sorry, this is wrong. As discussed in the mail thread

Reply:
> To clarify, if we want to adopt the "at least three months" policy, we will
need to wait for 3 months before upgrading to 1.85.

But this still holds, so the current solution in the PR is the best we can do.

Reply:
Hi, I'm a bit confused why we need this. I remember that we are using `minamal-version` to generate `Cargo.lock`?

Reply:
> But this still holds, so the current solution in the PR is the best we can do.

The last MSRV change was 7 months ago, I'm not sure why this holds.🤔 “At least three months” refers to either three months after the release of a new Rust version or three months after our last MSRV update?

> Hi, I'm a bit confused why we need this. I remember that we are using minimal-version to generate Cargo.lock?

But we call `cargo update faststr` later and it will upgrade the munge.

Reply:
Track in: #1000.